### PR TITLE
Add cookies page.

### PR DESCRIPTION
**Pull request by easternbloc**
Add Terms and conditions page.

Add links to both pages in the footer of the site.

### CODE REVIEW

**Comment by JoeChapman** [comment]
Where is the link to the cookies page?

**Comment by easternbloc** [comment]
Think that should be another task. We need to create a footer area for a link to the T&C also

File: views/terms.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{{<layout}}
+
+{{$header}}
+  Terms and conditions
+{{/header}}
+
+{{$content}}
+  <h2>How we use your information</h2>
+  <p>The data you provide is used in accordance with the <a href="https://www.gov.uk/government/organisations/home-office/about/personal-information-charter">Home Office's personal information charter</a>. This includes information about the applicant and any person sponsoring their application. The Home Office is the data controller, which means that it decides how this information will be used and stored. The information will be held in confidence and handled in accordance with the Data Protection Act 1998.</p>
```
Comment (JoeChapman) [inline views/terms.html]: the other external links have `rel="external"` attributes

Comment (easternbloc) [inline views/terms.html]: Good spot!

**Pull request merged**